source/reference/lists.txt

```diff
@@ -0,0 +1,140 @@
+..  _style-guide-lists:
+
+=====
+Lists
+=====
+
+.. default-domain:: mongodb
+
+You can create *ordered* or *unordered* lists in {+rst+}.
+
+Ordered Lists
+-------------
+
+Ordered lists show a number for each item in the list. Create an ordered list by
+starting a line with ``#.``, or by using a number followed by a ``.``. Using
+``#.`` in place of numbers prompts Snooty to generate list numbers automatically.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 50 50
+
+   * - Markup
+     - Result
+
+   * -
+
+       .. code-block:: rst
+          :copyable: false
+
+           #. Item 1
+           #. Item 2
+           #. Item 3
+
+     - #. Item 1
+       #. Item 2
+       #. Item 3
+
+.. list-table::
+   :header-rows: 1
+   :widths: 50 50
+
+   * - Markup
+     - Result
+
+   * -
+
+       .. code-block:: rst
+          :copyable: false
+
+           1. Item 1
+           2. Item 2
+           3. Item 3
+
+     - 1. Item 1
+       2. Item 2
+       3. Item 3
+
+Unordered Lists
+---------------
+
+Unordered lists show a bullet point for each item in the list. Create an
+unordered list by starting a line with ``*`` or ``-``. 
```

Comment:
Almost all of the unordered lists I see in the docs use `-`, not `*`. I noticed that you suggested all-#-sign lists first for ordered lists, a sensible default. I wonder if we should provide the `-` example first for unordered lists, since that's what writers generally use in docs.